InfoLogger/lib/api.js

```diff
@@ -164,9 +164,13 @@ module.exports.attachTo = (http, ws) => {
    * @param {Error} error
    * @param {number} status
    */
-  function handleError(res, error) {
+  function handleError(res, error, status) {
```

Comment:
Instead of using the if condition, you can mention a default value for status in the function header and it will automatically assign the value if the function call is not passing a parameter
```
  function handleError(res, error, status = 500) {
```

Reply:
Fixed it.